src/Form/RestaurantType.php

```diff
@@ -189,9 +189,8 @@ public function buildForm(FormBuilderInterface $builder, array $options)
 
                     if ($options['edenred_enabled']) {
                         $form
-                            ->add('edenredMerchantId', TextType::class, [
```

Comment:
Maybe we should still allow to set merchant ID manually, but it's fine. At least, we should show it (disabled).